File: src/packets/PacketC2SUpdate.js
const PacketC2S = require("./PacketC2S");

/**
 * Client -> Server
 * ID = 0x7D
 * Update
 * Request to start an update
 */
class PacketC2SUpdate extends PacketC2S {
    parseData() {}

    getID() {
        return PacketC2SUpdate.ID;
    }
}

PacketC2SUpdate.ID = 0x7D;

module.exports = PacketC2SUpdate;

File: src/RNetPi.Core/Packets/PacketC2SUpdate.cs
namespace RNetPi.Core.Packets;

/// <summary>
/// Client -> Server
/// ID = 0x7D
/// Update
/// Requests a software update
/// </summary>
public class PacketC2SUpdate : PacketC2S
{
    public const byte ID = 0x7D;

    public PacketC2SUpdate(byte[] data) : base(data)
    {
    }

    public override byte GetID() => ID;

    protected override void ParseData()
    {
        // No data to parse for update packet
    }
}